std/math.d

```diff
@@ -566,7 +566,7 @@ enum real SQRT1_2 =    SQRT2/2;                               /** $(SQRT)$(HALF)
  *
  * Params:
  *     Num = (template parameter) type of number
- *       x = real number value
+ *     _param_0 = real number value
```

Comment:
This doesn't work, and could also be confusing.  We would need to figure out how to support documenting nameless parameters in ddoc.